docs/.vuepress/config-extras.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
         "/reports/polaris",
         "/reports/trivy",
         "/reports/goldilocks",
+        "/reports/prometheus-collector",
```

Comment:
Could we change this to `resource-metrics`?

Reply:
Hmm yeah I think that'll be the link. Why did we go with `prometheus collector` in the UI rather than `resource metrics`? I think it's good, just curious